components/navigation/ui/src/main/resources/PhenoTips/LiveTableResults.xml

```diff
@@ -474,8 +476,46 @@
     ##
     ## Object properties from now on
     ##
-    ## Numbers, exact match
-    #elseif($propType == 'NumberClass' || $propType == 'BooleanClass')
+    ## Numbers, exact match or min/max filters
+    #elseif($propType == 'NumberClass')
+      #livetable_getTableName($colname)
+      #set($fromSql = "${fromSql}, ${tableName} ${safe_tableAlias}")
+      #set($whereSql = "${whereSql} and ${targetedObject}.id=${safe_tableAlias}.id.id and ${safe_tableAlias}.id.name = ?")
+      #set($discard = $whereParams.add("${colname}"))
+      ## Converts each used value into correct type
+      #set($numberType = $propClass.get($colname).getProperty('numberType').getValue())
+      #if($numberType == 'integer')
+        #set($typedFilterValue = $mathtool.toInteger($filterValue))
```

Comment:
I'm not convinced that rounding to the nearest integer is the best approach here. Again, it may be counterintuitive for users (who are not technical) and may wonder what kind of magic we're using to produce results. An exact match to the user input is probably what users would expect. If the user does not enter an integer value, then simply do not return any results.